frontend/src/types/pipeline/task.ts

```diff
@@ -19,7 +19,10 @@ export type TaskType =
   | "bb.task.database.create"
   | "bb.task.database.schema.update"
   | "bb.task.database.data.update"
-  | "bb.task.database.restore";
+  | "bb.task.database.restore"
+  | "bb.task.database.schema.update.ghost"
```

Comment:
"bb.task.database.schema.update.ghost" is changed to "bb.task.database.schema.update.ghost.sync"